packages/server/src/build.ts

```diff
@@ -36,9 +36,14 @@ export async function build(config: ServerConfig) {
   })
 
   await saveBlitzVersion(buildFolder)
-
   await nextBuild(nextBin, buildFolder, manifest, config)
 
+  try {
+    await startCustomServer(rootFolder, config, {watch, buildOnly: true})
+  } catch (e) {
+    console.log("No custom server found, proceeding")
+  }
```

Comment:
- I think we should first check if custom server exists instead of doing try/catch